UI/LocalizationSamples/RuntimeCultureSwitching/UnoLocalization.Shared/LanguageSettings.xaml.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+
+// The Blank Page item template is documented at https://go.microsoft.com/fwlink/?LinkId=234238
+
+namespace UnoLocalization
+{
+	/// <summary>
+	/// An empty page that can be used on its own or navigated to within a Frame.
+	/// </summary>
+	public sealed partial class LanguageSettings : Page
+	{
+		public LanguageSettings()
+		{
+			this.InitializeComponent();
+		}
+
+		private void SetAppLanguage(object sender, RoutedEventArgs e)
+		{
+			if ((sender as Button)?.Tag is string tag)
+			{
+				// Change the app language
+				var culture = new System.Globalization.CultureInfo(tag);
+
+#if WINDOWS_UWP
+				Windows.Globalization.ApplicationLanguages.PrimaryLanguageOverride = culture.IetfLanguageTag;
```

Comment:
Isn't this working in 3.6 ? @carldebilly  you added support for this, right ?

Reply:
@jeromelaban It should be, but this is using 3.5.1 atm.
I wasn't sure about to complicating the guide with package update stuffs. But, I guess 3.6.0 is on the horizon?

Reply:
The plan is to push the new docs content at the same time as 3.6 is released ;)